docs/jwst/residual_fringe/main.rst

```diff
@@ -27,8 +26,6 @@ The ``residual_fringe`` step can accept several different forms of input data, i
 
   - an association table (in json format) containing a list of input files
```

Comment:
Is it still true that this step can accept an ASN table with a list of input files?  Or just a single input file/datamodel?